modules/transport-netty4/src/internalClusterTest/java/org/elasticsearch/http/netty4/Netty4ChunkedContinuationsIT.java

```diff
@@ -317,20 +315,14 @@ public void onFailure(Exception exception) {
 
     private static Releasable withResourceTracker() {
         assertNull(refs);
-        final ReachabilityChecker reachabilityChecker = new ReachabilityChecker();
         final var latch = new CountDownLatch(1);
-        refs = LeakTracker.wrap(reachabilityChecker.register(AbstractRefCounted.of(latch::countDown)));
+        refs = AbstractRefCounted.of(latch::countDown);
```

Comment:
Are you sure it's not worth leaving the leak tracking in? it did make it much easier to troubleshoot.

Reply:
I'm not sure, no, although I'd generally rather keep debug stuff out of tests while they're not known to be flaky. If we keep having problems here then we can put it back easily enough.